README.md

```diff
@@ -31,83 +33,140 @@ The extension provides a fast and accurate solution for:
-The Liver segments can be defined using the following steps:
-
-1.  Select `Vascular Territory Segmentation`
-2.  Select the `Segmentation`.
-3.  You can hide the liver segmentation for better visibility in the 3D view by first selecting the liver segmentation, then select `Hide`.
-4.  Select the hepatic/portal segmentation for `Segment`. The 3D view will then automatically switch to landmark "Place Mode" 
-and a `Vessel points` list will be automatically generated.
-5.  Place user-defined markup landmarks along the selected hepatic/portal model in 3D View to mark the part of the vessel where centerlines should be generated.
-6.  Once all the points are placed, Click `Add Vessel Centerline`.
-7.  Optionally, repeat steps 4-6 for generation of additional centerline for other vessel segments (portal/hepatic).
-8.  For generation of several vascular territory segments, select `Vascular Territory` and "Create new territory ID".
-9.  Repeat steps 4 to 7 for creating centerlines to be used for calculation of the new Vascular Territory.
-10.  Click on `Calculate Vascular Territory Segmentation`. The generated vascular territories will then be visible in the 3D- and 2D Views.
-11. Multiple sets of vascular territories can be generated and stored in separat Vascular Territory Segmentations.
-To generate a new set of vascular territories, select "Create New Vascular Territory Segmentation" in `Vascular Territory Segmentation`. 
-Then repeat steps 2-9.
-
-![Liver segments -- placing fiducials](https://github.com/ALive-research/Slicer-Liver/blob/master/Screenshots/Slicer-Liver_screenshot_06.png?raw=true)
-
-![Liver segments -- placing fiducials](https://github.com/ALive-research/Slicer-Liver/blob/master/Screenshots/Slicer-Liver_screenshot_08.png?raw=true)
-
- ![Slicer-Liver_screenshot_14](https://github.com/dalbenzioG/Slicer-Liver/blob/master/Screenshots/Slicer-Liver_screenshot_14.jpg?raw=true)
+Our method for liver segment classification involves defining segments using centerlines created from user-specified points. These centerlines serve as the foundation for computing liver segments within the image space. The computation leverages shortest-distance mapping
+
+1. **Vascular Territory Segmentation**:
+   - Create or Select a vascular territory segmentation from the dropdown menu.
+2. **Vascular Territory**:
+   - Create a new territory ID.
+3. **Segmentation**:
+   - Select the segmentation node representing the hepatic/portal vessels.
+4. **Hide Unnecessary Segments**:
+   - Use the `Show/Hide` button to hide the liver and/or tumor segmentation nodes if they obstruct the view. This step is not required for creating centerlines on vessel branches but can improve visibility.
+5. **Vessel Points**:
+   - Create a new Point List for `Vessel points`.
+   - Click the arrow button next to `Vessel points` and place fixed landmark points on the hepatic/portal segmentation. These points will be used to extract the centerlines of user-defined vessel branches.
```

Comment:
(The arrow button is also "automatically" clicked when selecting hepatic/portal in step 3)